e2e/kind-cluster-config.yaml

```diff
@@ -0,0 +1,28 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker 
+  labels:
+    worker: true
```

Comment:
you generally dont need this because control-plane nodes have taints to prevent scheduling pods on them

Reply:
Kind of, yes, but I also need to explicitly wait after all nodes are ready (`kubectl wait -l ...`), and I think it's better to have an explicit label to wait on rather than relying on some default node label that may randomly change in the future